src/openfl/display/Stage.hx

```diff
@@ -116,7 +116,7 @@ class Stage extends DisplayObjectContainer implements IModule {
 	public var fullScreenWidth (get, never):UInt;
 	public var quality:StageQuality;
 	public var scaleMode:StageScaleMode;
-	public var showDefaultContextMenu:Bool;
+	public var showDefaultContextMenu (default, set):Bool;
```

Comment:
This looks weird, the property has `default` read access, but it's never actually read. It should be at least `never` then, or it should have a `get` that proxies the `window` field value to be consistent. Another option is to just have the `setShowDefaultContextMenu` (or something) method and don't mess with properties at all.